packages/rich-text-structure/package.json

```diff
@@ -0,0 +1,27 @@
+{
+	"name": "@wordpress/rich-text-structure",
+	"version": "1.0.0",
```

Comment:
It should be `1.0.0-beta.0` or anything < than `1.0.0` to let `Lerna` bump it to `1.0.0`.

Reply:
Why should it be possible to bump to 1.0.0?